app/build.gradle.kts

```diff
@@ -23,6 +23,10 @@ android {
     }
 
     sourceSets { map { it.java.srcDir("src/${it.name}/kotlin") } }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
+    }
```

Comment:
@android10 is there a better place to put this? I couldn't put it under `compilation.gradle.kts` because it requires kotlin plugin.

Reply:
Could we not add the kotlin plugin to the `buildSrc/build.gradle.kts` dependencies to allow for this to be used within the scripts? (also could you make the `Dependencies` object private if we do decide to add this there inside that `build.gradle`)

Reply:
I need some guidance from Fernando on this. I couldn't even create a .gradle.kts file :D

Reply:
@Gizem I would not start mixing things up. AFAIK we do not need this right? By default it compiles against Java 8. 
In case you need it, maybe I can raise a PR tomorrow and provide more context but I would extract all this and keep the ```build.gradle.kts``` as small and clean possible. 

@BradleyWire I do not understand your point. "Dependencies" cannot be private. ```buildSrc``` is a separated project that builds stuff up, which becomes available to the rest of gradle scripts.

Reply:
@android10 `Dependencies` is the object inside your `build.gradle.kts` inside the `buildSrc` module, it doesn't need to have public access seeing as it's only constant belongs in the same file as it's own 

```
object Dependencies {
    const val AndroidBuildTools = "com.android.tools.build:gradle:4.0.0"
}
```

I'd be safe in assuming this only need to be private to the file. It's a minor cosmetic change and is out of the scope of this PR which is why I asked to only change it if you're adding a dependency change to the gradle file.

Reply:
Ok. I thought you were talking about something else. 
Here is the fix:
https://github.com/wireapp/wire-android-reloaded/pull/12/files

Reply:
Thanks @android10, Also, you'll need the `kotlinOptions` closure eventually otherwise you'll run into:

> Cannot inline bytecode built with JVM target 1.8 into bytecode that is being built with JVM target 1.6

I've ran into with this previously with the `observe(..)` extension function when observing a `LiveData` instance (which is heavily used in our architecture)

On the topic of where it belongs though, to me I understand the point of keeping the scripts concise and simple, but, It'd also make sense to have all the options for compilation in the same script to keep in line with single responsibility principle.